Resources/Prototypes/Entities/Objects/Weapons/Guns/Projectiles/projectiles.yml

```diff
@@ -721,3 +721,97 @@
             - Impassable
             - HighImpassable
     - type: GrapplingProjectile
+
+- type: entity
+  id: BaseBulletEmp
+  name: base bullet emp
+  parent: BaseBulletTrigger
+  noSpawn: true
+  components:
+  - type: Projectile
+    damage:
+      types:
+        Shock: 0
+    layers:
```

Comment:
i think you're missing the proper sprite component here and the yml linter caught it. `layers: ` should be a property of a `- type: Sprite` and also with a `sprite: <path to sprite>`